modules/services/elasticsearch/main.tf

```diff
@@ -114,3 +119,27 @@ resource aws_security_group_rule "es-edxapp-outbound-rule" {
   protocol = "all"
   cidr_blocks = ["0.0.0.0/0"]
 }
+
+resource aws_cloudwatch_log_group "openedx" {
+  name = "openedx-cloudwatch-log-group"
```

Comment:
@Cup0fCoffee Maybe we shoudl rename this to "${var.customer_name}-${var.environment}-elasticsearch-cloudwatch-log-group" OR "${var.customer_name}-${var.environment}-cloudwatch-log-group", what do you think?

Reply:
As per offline discussion we are going to use the first option `"${var.customer_name}-${var.environment}-elasticsearch-cloudwatch-log-group"`

/cc @Cup0fCoffee

Reply:
Done!